Gemfile

```diff
@@ -29,7 +29,7 @@ gem 'bootstrap-kaminari-views'
 gem 'haml-rails'
 
 # Processing
-gem 'audio_monster'
+gem 'audio_monster', git: 'https://github.com/PRX/audio_monster'
```

Comment:
currently pointing at GH till 1.3.0 is on rubygems.org

Reply:
done! https://rubygems.org/gems/audio_monster/versions/1.3.0